test/integration/connect/envoy/socat-1.7.3.0-windows/README.md

```diff
@@ -0,0 +1,9 @@
+Since no one else has compiled socat for windows, here it is.
+
+I used some optimizations, which should help this version of socat stand 
```

Comment:
In my opinion we should avoid comments in the first person.

Reply:
I delete the file since this is from the original repository